sway-core/src/asm_generation/mod.rs

```diff
@@ -1174,10 +1174,11 @@ fn build_contract_abi_switch(
         });
     }
 
-    // if none of the selectors matched, then ret
+    // if none of the selectors matched, then revert
     asm_buf.push(Op {
         // see https://github.com/FuelLabs/sway/issues/97#issuecomment-875674105
-        opcode: Either::Left(VirtualOp::RET(VirtualRegister::Constant(
+        // and https://github.com/FuelLabs/sway/issues/444#issuecomment-1012507337
+        opcode: Either::Left(VirtualOp::RVRT(VirtualRegister::Constant(
             ConstantRegister::Zero,
         ))),
         comment: "return if no selectors matched".into(),
```

Comment:
Should this comment now say "revert if no selectors match" ?

Reply:
yessir